src/main/java/com/product/model/CustomerProductResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.product.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class CustomerProductResponse {
+
+	Customer customerResponse;
```

Comment:
why call it `customerResponse` and product one just `product`

Be careful with the variable naming here, because that is what you will see when you do the REST call from Postman


If you want to keep the variable name, but in json response you want a different name then read into @JsonProperty annotation
Read : https://stackoverflow.com/questions/12583638/when-is-the-jsonproperty-property-used-and-what-is-it-used-for

Reply:
Read and done the changes.